src/frontend/.env.example

```diff
@@ -7,4 +7,8 @@ NEXT_PUBLIC_UNCONNECTED_NOTICE_MESSAGE='This is a testnet interface. All coins a
 NEXT_PUBLIC_READ_ONLY=0
 NEXT_PUBLIC_READ_ONLY_MESSAGE=''
 NEXT_PUBLIC_DEFAULT_MARKET_ID=8
+# You can specify an integrator address below. Make sure that you've
```

Comment:
How about updating the README for this?

I think the docs site will also need to be updated (should be a separate linear task)